docs/docs/node-api-helpers.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: Node API Helpers
-description: Documentation on API helpers for creating nodes within Gatsby's GraphQL data layer
+title: Punkty funkcji pomocniczych API
+description: Dokumentacja na temat API funkcji pomocniczych do tworzenia punktów Gatsby'ego poprzez warstwę z danymi GraphQL
```

Comment:
We wszystkich innych tłumaczeniach, nie odmieniamy słowa `Gatsby`. Czyli nie `punktów Gatsby'ego`, a po prostu `punktów Gatsby` ;). I może lepiej przetłumaczyć `data layer` na `warstwę danych`? I całe zdanie brzmi nieco nienaturalnie... :)

```suggestion
description: Dokumentacja API funkcji pomocniczych do tworzenia punktów Gatsby wewnątrz warstwy danych GraphQL.
```

Reply:
> We wszystkich innych tłumaczeniach, nie odmieniamy słowa `Gatsby`. Czyli nie `punktów Gatsby'ego`, a po prostu `punktów Gatsby` ;).  

Uważam, że słowo `Gatsby` jak najbardziej powinno zostać odmieniane, ponieważ brzmi to dużo naturalniej.

> We wszystkich innych tłumaczeniach, nie odmieniamy słowa `Gatsby`.

Już jedno moje tłumaczenie zostało zaakceptowane przez kogoś innego, które zawierało pełną odmianę słowa `Gatsby`.

Reply:
Hm, to ciekawe. Moje PR'y z odmianą nie przeszły, co nie zmienia faktu że również wydaje się dla mnie ona bardziej naturalna. Btw, jeszcze jedna uwaga - bazując na informacjach w pliku `style-guide.md`, nie tłumaczymy słowa `Node` - zostawiamy je po angielsku, ale odmieniamy po polsku :)

Reply:
Zgadzam się. Jak dobrze kojarzę, też poprawiłem kiedyś na jednolite. Z tego co pamiętam nie odmienialiśmy wyrazu - tu istotna byłaby spójność. Całe szczęście to będą proste poprawki ^ Odnośnie warstwy danych: +1. Jasno wynika jakie powinno być tłumaczenie tego.

Reply:
* data layer =! layer with data

Reply:
Jeśli chodzi o odmianę Gatsby to zdecydowanie bez odmiany :) @jakex7 w Twoim PR musieliśmy to przegapić.